docs/setup-sensor/podman.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Installing Orb using Podman Quadlet
+shortTitle: Podman Quadlet
+metaDescription: Run the Orb sensor as a Podman container managed by a systemd quadlet on any compatible system.
+section: setup-sensor
+layout: guides
+imageUrl: ../../images/devices/podman.png
+subtitle: 'Difficulty: Intermediate 🧑‍🔬'
+---
+
+# Installing Orb using Podman Quadlet
+
+## Introduction
+
+[Podman](https://podman.io/) is a daemonless container engine that runs containers in isolated environments. This guide shows you how to run the Orb sensor as a Podman container managed by a systemd quadlet, a declarative way to define and manage containers as system services. This method is ideal for Linux systems where systemd is available, such as servers, firewalls, routers, or personal computers with Podman installed.
+
+This guide assumes you have Podman already installed and running on your host system. All commands assume being ran as the root user.
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+- A Linux host machine with Podman installed (version 4.0 or later for quadlet support). See the official [Podman installation guides](https://podman.io/getting-started/installation).
+  - Install on Fedora: `dnf install podman`
+  - Install on Ubuntu: `apt-get install podman`
+- Systemd installed and running (standard on most Linux distributions).
+- Basic familiarity with using the command line or terminal on your host system.
+- Your host machine must be connected to the network you wish to monitor.
+- Rootful Podman, as the Orb sensor requires `CAP_NET_RAW` for network monitoring.
+
+Verify Podman installation:
+
+```bash
+podman --version
+```
+
+## Quick install
+
+If you'd like to run Orb with auto-updates and don't want to get into the details, run these commands:
+
+```bash
+mkdir -p /etc/containers/systemd
+curl -fsSL https://orb.net/docs/scripts/podman/orb-sensor.container -o /etc/containers/systemd/orb-sensor.container
```

Comment:
this file doesn't exist yet and could be added to this PR in the right place, it would be the same as what is pasted below